IPython/utils/_process_cli.py

```diff
@@ -11,16 +11,11 @@
 in the details. 
 """
 
-# Import cli libraries:
-import clr
```

Comment:
Let's leave this alone; it's possible that it needs to import this first to use CLI libraries. I don't know much about how IronPython works, and our CI doesn't cover it.